<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>722</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="logo">
          <property name="minimumSize">
           <size>
            <width>291</width>
            <height>291</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>291</width>
            <height>291</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap>media/logo.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>181</width>
            <height>23</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>23</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Inter SemiBold</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">margin: 0px 0px 10px 0px;</string>
          </property>
          <property name="text">
           <string>Ciudad de origen:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="listaOrigen">
          <property name="minimumSize">
           <size>
            <width>181</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Inter SemiBold</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <item>
           <property name="text">
            <string>Catamarca</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>C칩rdoba</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>La Rioja</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Tucum치n</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="verListaClientes">
        <property name="minimumSize">
         <size>
          <width>181</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>181</width>
          <height>31</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Inter SemiBold</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">#verListaClientes {
	background-color: #2e58ff;
	border-radius: 10%;
	color: #ffffff;
	border: none;
}

#verListaClientes::hover {
	background-color: #172e87;
}</string>
        </property>
        <property name="text">
         <string>Ver lista de clientes</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_3">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>61</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>61</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="verViajesDisponibles">
        <property name="minimumSize">
         <size>
          <width>181</width>
          <height>51</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>181</width>
          <height>51</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Inter SemiBold</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">#verViajesDisponibles {
	background-color: #2e58ff;
	border-radius: 10%;
	color: #ffffff;
	border: none;
}

#verViajesDisponibles::hover {
	background-color: #172e87;
}</string>
        </property>
        <property name="text">
         <string>Comprar pasaje</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>61</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>61</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="minimumSize">
           <size>
            <width>181</width>
            <height>23</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>23</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Inter SemiBold</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">margin: 0px 0px 10px 0px;</string>
          </property>
          <property name="text">
           <string>Ciudad de destino:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="listaDestino">
          <property name="minimumSize">
           <size>
            <width>181</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Inter SemiBold</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <item>
           <property name="text">
            <string>Catamarca</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>C칩rdoba</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>La Rioja</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Tucum치n</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
